docs/technical-reference.md

```diff
@@ -345,6 +348,11 @@ Use current template > Next > Specify stack details), set the following paramete
 * *AuthClientId*: *Client ID*
 * *AuthBaseUrl*: *Issuer URL V2*
 
+#### Setting the default role
+**The Quilt admin must log in and set the default role** in order for new users to be able to use Quilt.
+Without this, new users will have no permissions.
+![](imgs/default-role.png)
+
```

Comment:
What's wrong with this?
https://github.com/quiltdata/quilt/blob/49bbe650aa7e6c97ae82ea24b8a79b1861175d4d/docs/technical-reference.md?plain=1#L253-L260

Reply:
I have removed the duplication in favor of the SSO section, where it makes the most sense and can be addressed sequentially after deployment.

Reply:
Well, I'm not sure that's right, because it also affects password signup like on open.